src/plugins/intel_cpu/tests/functional/subgraph_tests/src/conv_sum_broadcast.cpp

```diff
@@ -241,11 +250,11 @@ class ConvSumInPlaceTestInt8 : public ConvSumInPlaceTest {
 
     void SetUp() override {
         abs_threshold = 1.001f;
-        using ngraph::pass::ConvertPrecision;
+        using ov::pass::ConvertPrecision;
         ConvSumInPlaceTest::SetUp();
         functionRefs = function->clone();
-        ngraph::pass::ConvertPrecision<ngraph::element::Type_t::i8, ngraph::element::Type_t::f32>().run_on_model(functionRefs);
-        ngraph::pass::ConvertPrecision<ngraph::element::Type_t::u8, ngraph::element::Type_t::f32>().run_on_model(functionRefs);
+        ngraph::pass::ConvertPrecision<ov::element::Type_t::i8, ov::element::Type_t::f32>().run_on_model(functionRefs);
```

Comment:
May we try `ov::pass::ConvertPrecision` (from transformations/convert_precision.hpp) instead?

Reply:
Not sure whether `ov::pass::ConvertPrecision` is equal to  `ConvertConstantsPrecision, ConvertParametersPrecision, ConvertParametersPrecision`, maybe we can port `ngraph::pass::ConvertPrecision` to here?

Reply:
If I understand it right, these passes are moved to tests utilities just because original legacy passes were cleaned up, but these were hard to remove because of it's usage in tests. As I can see, it's also used in `LayerTestsCommon::ConvertRefsParams()` (legacy tests class), which equivalent is `SubgraphBaseTest::update_ref_model()` (new tests class) with `ov::pass::ConvertPrecision` (+ @iefode )

So let's try to get rid of legacy pass same way

Reply:
Seems we cannot directly apply `ov::pass::ConvertPrecision`, see blow code block:

```
       pass::Manager manager;
       precisions_map convert_precisions = {{ov::element::Type_t::i8, ov::element::Type_t::f32},
                                             {ov::element::Type_t::u8, ov::element::Type_t::f32}};
       manager.register_pass<ov::pass::ConvertPrecision>(convert_precisions, type_to_fuse_map{}, false, false);
       manager.run_passes(functionRefs);
       functionRefs->validate_nodes_and_infer_types();
```

It will report don't support "conversion from i8 to f32":

```
[ RUN      ] smoke_Conv_Sum_Broadcast_INT8/ConvSumInPlaceTestInt8.CompareWithRefs/IS=[?.32.?.?][1.64.8.8]_TS=((1.32.10.10)(1.32.10.10)(1.32.10.10)(1.32.3.3)(1.32.3.10))_((1.64.8.8))_bias=False_Fused=Divide(PerChannel).Subtract(PerChannel).FakeQuantize(PerChannel)

MEM_USAGE=625772KB
Access PluginCache ov core. OV Core use count: 0
Created ov core.
unknown file: Failure
C++ exception with description "Exception from src/common/transformations/src/transformations/convert_precision.cpp:1273:
Precision conversion from i8 to f32 is not supported
" thrown in SetUp().
Reset PluginCache. OV Core use count: 1
[  FAILED  ] smoke_Conv_Sum_Broadcast_INT8/ConvSumInPlaceTestInt8.CompareWithRefs/IS=[?.32.?.?][1.64.8.8]_TS=((1.32.10.10)(1.32.10.10)(1.32.10.10)(1.32.3.3)(1.32.3.10))_((1.64.8.8))_bias=False_Fused=Divide(PerChannel).Subtract(PerChannel).FakeQuantize(PerChannel), where GetParam() = (({ ?, 32, ?, ? }, { { 1, 32, 10, 10 }, { 1, 32, 10, 10 }, { 1, 32, 10, 10 }, { 1, 32, 3, 3 }, { 1, 32, 3, 10 } }), ({ 1, 64, 8, 8 }, { { 1, 64, 8, 8 } }), false, (16-byte object <00-A6 BD-8E 36-56 00-00 F0-A5 BD-8E 36-56 00-00>, { "FakeQuantize" }), {}) (17 ms)
```

Here test cases maybe only want to change constant and parameter node's precision, but I don't understand why change i8/u8 to fp32, maybe i32 is more reasonable?